lib/paginator/ecto/query/helpers.ex

```diff
@@ -0,0 +1,35 @@
+defmodule Paginator.Ecto.Query.Helpers do
```

Comment:
Can this module have a more specific name? e.g. Query.FieldOrExpression?

Reply:
Updated the name to be `FieldOrExpression`